### PR TITLE
Better handle liker.land sign in

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: app/i18n/en.json [hunk 1]
```diff
@@ -110,6 +110,7 @@
     "errorLikeCo": "Error occurs when signing in to like.co",
     "errorLikerLand": "Error occurs when signing in to liker.land",
     "heading": "At Republic of Liker Land, every Like is a reward to creators",
+    "LoadingTakesLonger": "This is taking longer than usual.",
```
Comment (williamchong007) [inline app/i18n/en.json:113]: `This is taking longer than usual...`

File: app/i18n/zh-Hant-HK.json [hunk 2]
```diff
@@ -213,6 +213,7 @@
     "errorLikeCo": "登入 like.co 錯誤",
     "errorLikerLand": "登入 liker.land 錯誤",
     "heading": "讚賞公民共和國，每個 Like 都是對創作者的實質回報",
+    "LoadingTakesLonger": "這看來比平常花更長的時間。",
```
Comment (williamchong007) [inline app/i18n/zh-Hant-HK.json:216]: `讀取需時，請稍等...`

Comment (nwingt) [reply app/i18n/zh-Hant-HK.json]: Updated

File: app/i18n/en.json (hunk 1 above)
Comment (nwingt) [reply app/i18n/en.json]: Updated

**Pull request merged**